### PR TITLE
[rc] Angular 9; drop RxJS 5

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**
Depends on: https://github.com/blackbaud/skyux-sdk-builder/pull/212

FYI: The documentation should fail because it's using an older version of Popovers (once this is released, it should work).

### CODE REVIEW

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux-popovers?_sp.spa=pr-rc-angular-9](https://developer.blackbaud.com/skyux-popovers?_sp.spa=pr-rc-angular-9)

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux-popovers?_sp.spa=pr-rc-angular-9](https://developer.blackbaud.com/skyux-popovers?_sp.spa=pr-rc-angular-9)

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux-popovers?_sp.spa=pr-rc-angular-9](https://developer.blackbaud.com/skyux-popovers?_sp.spa=pr-rc-angular-9)

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux-popovers?_sp.spa=pr-rc-angular-9](https://developer.blackbaud.com/skyux-popovers?_sp.spa=pr-rc-angular-9)

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux-popovers?_sp.spa=pr-rc-angular-9](https://developer.blackbaud.com/skyux-popovers?_sp.spa=pr-rc-angular-9)

File: src/app/docs/dropdown/dropdown-docs.component.ts [hunk 1]
```diff
@@ -67,4 +67,8 @@ export class DropdownDocsComponent {
       this.demoSettings.style = change.style;
     }
   }
+
+  public actionClicked(name: string): void {
+    console.log('Action clicked:', name);
+  }
```
Comment (Blackbaud-AlexKingman) [inline src/app/docs/dropdown/dropdown-docs.component.ts]: Noticed this slipped in there. Is this deliberate?

File: src/app/public/modules/popover/popover.component.ts [hunk 2]
```diff
@@ -173,11 +180,14 @@ export class SkyPopoverComponent implements OnInit, OnDestroy {
...
     private elementRef: ElementRef,
-    private windowRef: SkyWindowRefService
+    private windowRef: SkyAppWindowRef
   ) { }
 
   public ngOnInit(): void {
     this.preferredPlacement = this.placement;
+  }
+
+  public ngAfterViewInit(): void {
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/popover/popover.component.ts]: Why did you move this to `AfterViewInit`?

Comment (Blackbaud-SteveBrush) [reply src/app/public/modules/popover/popover.component.ts]: Angular 9 updated `ViewChild` to require we set a `static` property. The default value is `static: false`, which means that the element is only accessible in the `ngAfterViewInit`. Angular had some mechanisms in place before version 9 that would allow for accessing children in `ngOnInit`, but now we'd have to declare the child `static: true`, which I'm uncomfortable doing since it would be new behavior. I'm open to changing it in a future pull request but for now I wanted to maintain existing functionality. Good eye!

File: src/app/docs/dropdown/dropdown-docs.component.ts (hunk 1 above)
Comment (Blackbaud-SteveBrush) [reply src/app/docs/dropdown/dropdown-docs.component.ts]: The demo had a reference to the `actionClicked` callback but the method didn't exist on the class (throwing template check errors). I added the class with a console log so that consumer could see it doing something. I'm happy to remove it or change it if you have something else in mind.

Comment (Blackbaud-AlexKingman) [reply src/app/docs/dropdown/dropdown-docs.component.ts]: Nope, that works. Thanks for taking care of it.

File: src/app/public/modules/popover/popover.component.ts (hunk 2 above)
Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/popover/popover.component.ts]: Ok, thanks for the explanation. Considering the alternatives, I think what you did here is probably the safest route. 👍
